### PR TITLE
Correct for orientation of Aruco markers with ID:63

### PR DESCRIPTION
**Pull request by SamCWill**
# Correct for orientation of Aruco markers with ID:63
## What?
For Aruco markers of ID:63, measure reflectance panel DN beneath the marker instead of to its left.
## Why?
When measuring mean reflectance panel DN, it is assumed that the reflectance panel is to the left of the Aruco marker (when the image is rotated such that the Aruco marker is facing top-up). For reflectance panels with Aruco markers of ID:63, this is not the case. They are oriented such that their reflectance panels are beneath their Aruco markers. This change accounts for that.
## PR Checklist
- [X] Merged latest master
- [X] Updated version number
- [X] Version numbers match between package ``__version__`` and *pyproject.toml*
- [ ] All private git packages are at their newest version in both *pyproject.toml* and *environment.yml*
- [ ] All git package version numbers match between *pyproject.toml* and *environment.yml*
- [X] dist/ImageryCorrector.exe has been rebuilt
## Breaking Changes

### CODE REVIEW

File: imgcorrect/detect_panel.py [hunk 1]
```diff
@@ -70,9 +70,17 @@ def extract_panel_bounds(image):
...
         logger.debug("Calibration image GSD: %10.5f m/pixel", gsd)
 
+        # by default, expects panel to left of aruco marker
         top_aruco_line = corners[0][0][0] - corners[0][0][3]
         top_aruco_line_middle = top_aruco_line / 2.0 + corners[0][0][3]
 
+        # panels with markers of id:63 are incorrectly oriented
+        # in this case, panel is beneath marker
+        if ids[0][0] == 63:
+            logger.info("Aruco ID: 63 detected. Adjusting for panel beneath marker.")
```
Comment (dzermas) [inline imgcorrect/detect_panel.py:80]: This is the right side of the aruco code, right?

Comment (SamCWill) [reply imgcorrect/detect_panel.py:80]: I'm not sure I understand the question. For images with only one aruco marker, `ids` will look something like this: `[[23]]`

Comment (SamCWill) [reply imgcorrect/detect_panel.py:80]: Ah ok. I believe the order of the corners is top-left, top-right, bottom-right, bottom-left. This is based on this description of `corners` from opencv documentation: 

> vector of detected marker corners. For each marker, its four corners are provided, (e.g std::vector<std::vector<cv::Point2f> > ). For N detected markers, the dimensions of this array is Nx4. The order of the corners is clockwise.

So the line from 000 to 003 is the left side of the marker, and 003 to 002 is the bottom. This lines up with the image @BrianEickhoff provided: 
image

Comment (dzermas) [reply imgcorrect/detect_panel.py:80]: Yes, this makes sense. Thank you!

**Review by dzermas — APPROVED** [review]
LGTM

**Pull request merged**